SETUP/upgrade/15/20201217_alter_user_profiles.php

```diff
@@ -0,0 +1,23 @@
+<?php
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Changing default on user_profiles.i_newwin..\n";
+$sql = "
+    ALTER TABLE user_profiles
```

Comment:
just apply this for everyone please :D.

Reply:
Tempting, but I suspect some long-time users would object to someone changing their settings on them like this :D